ensawards.org/src/components/atoms/SmartContractMetadata.tsx

```diff
@@ -94,72 +82,60 @@ export function SmartContractMetadata({ metadata, name }: SmartContractMetadataP
         </GenericTooltip>
         <GenericTooltip
           content={
-            <div className="w-full max-w-[275px] flex flex-col gap-1 justify-start items-start">
+            <div className="w-full max-w-[275px] flex flex-col gap-2 justify-start items-start">
               <div className="self-stretch flex flex-row flex-nowrap justify-between items-center">
-                <h4 className="text-base font-semibold">avatar</h4>
-                <span
-                  className={cn(
-                    "text-xs rounded-full py-1 px-2",
-                    metadata && metadata.avatar
-                      ? "text-emerald-100 bg-emerald-600"
-                      : "text-red-100 bg-red-600",
-                  )}
-                >
+                <h4 className="text-sm leading-normal font-semibold">avatar</h4>
+                <span className="text-xs leading-normal font-semibold rounded-md py-0.5 px-2 bg-white text-black">
                   {metadata && metadata.avatar ? "Set" : "Not set"}
                 </span>
               </div>
-              <span className="w-full h-[1px] self-stretch bg-white" />
-              <p>Customize an avatar image for the contract.</p>
-              <a
-                className="text-blue-400 whitespace-nowrap hover:underline hover:underline-offset-[25%]"
-                href="https://discuss.ens.domains/t/ensip-proposal-contract-metadata-standard-and-text-records/21397"
-                target="_blank"
-                rel="noreferrer noopener"
-              >
-                Learn more
-              </a>
+              <div className="flex flex-col gap-1 justify-start items-start">
+                <p>Customize an avatar image for the contract.</p>
```

Comment:
```suggestion
                <p>Customize an avatar image for the contract that could appear in apps such as wallets and block explorers.</p>
```

Goal: This idea just came to me and I think it will help better communicate the value prop.